setup.py

```diff
@@ -6,15 +6,17 @@
 import sys
 
 file_path = os.path.dirname(os.path.realpath(__file__))
-ganga_python_dir = os.path.join(file_path, 'python')
```

Comment:
Perhaps, yes. I was partly trying to clear up the global namespace. Significantly though, the `_get_test_env` function will likely be able to be removed soon since pytest is a little cleverer about setting `PYTHONPATH` etc.